### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by wuchencm** [apache/skywalking#8265] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Helm Chart (apache/skywalking-kubernetes)

### What happened

I install skywalking with helm chat, now the skywalking ui is nothing when i browse it.
 
skywalking-ui 
version :skywalking-ui:8.8.1 
env: SW_OAP_ADDRESS  http://skywalking-oap:12800     

### What you expected to happen

install skywalking with helm chat

### How to reproduce

always

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Helm repo has its e2e already. Don't think this is a bug.

**Reply by wuchencm** [reply]
I haved try again, its ok with ingress now
